validationFunctions/validateProcessingActivities.test.js

```diff
@@ -0,0 +1,93 @@
+const validateProcessingActivities = require("./validateProcessingActivities");
+const {
+  processingActivitiesEnum
+} = require("../enums/processingActivitiesEnum");
+
+describe("Function: validateProcessingActivities", () => {
+  it("should return false when input is not an array", () => {
+    // Arrange
+    const badInputs = [null, undefined, {}, "", 123];
+
+    // Act & Assert
+    badInputs.forEach((input) => {
+      const valid = validateProcessingActivities(input);
+      expect(valid).toBe(false);
+    });
+  });
+
+  it("should return false when input is an empty array", () => {
+    // Arrange
+    const input = [];
+
+    // Act
+    const valid = validateProcessingActivities(input);
+
+    // Assert
+    expect(valid).toBe(false);
+  });
+
+  it("should return false when 'NONE' or 'DONT_KNOW' are combined with other values", () => {
+    // Arrange
+    const badCombinations = [
+      [processingActivitiesEnum.NONE.key, "VACUUM_PACKING"],
+      [processingActivitiesEnum.DONT_KNOW.key, "SOUS_VIDE"],
+      [
+        processingActivitiesEnum.NONE.key,
+        processingActivitiesEnum.DONT_KNOW.key,
+        "VACUUM_PACKING"
+      ]
+    ];
+
+    // Act & Assert
+    badCombinations.forEach((combination) => {
+      const valid = validateProcessingActivities(combination);
+      expect(valid).toBe(false);
+    });
+  });
+
+  it("should return true when 'NONE' or 'DONT_KNOW' are the only selected values", () => {
+    // Arrange
+    const invalidSelections = [
+      [processingActivitiesEnum.NONE.key],
```

Comment:
any reason for using `.key` here and in other places but also using raw text value below e.g "VACUUM_PACKING"?